test/foundry/zone/PreAndPostFulfillmentCheck.t.sol

```diff
@@ -491,7 +491,7 @@ contract PreAndPostFulfillmentCheckTest is BaseOrderTest {
         });
 
         // assertions
-        assertTrue(statefulZone.authorizeCalled());
+        // assertTrue(statefulZone.authorizeCalled());
```

Comment:
how come these ones are commented out?

Reply:
Dates back to when optimized wasn't calling authorize, updating now.